tests/test_local_directory.py

```diff
@@ -0,0 +1,61 @@
+from _pytest.monkeypatch import monkeypatch
+import pytest
+from pathlib import Path, PosixPath
+
+from stores.directory.local import LocalDirectoryStore
+
+def test_local_directory_store_path():
+    """
+    Ensures a LocalDirectoryStore can be instantiated from a
+    given Path and that it can store the path to a correct 
+    existing local directory.
+    """
+
+    test_path = Path("tests/test_local_directory.py")
```

Comment:
needs to. be a directory, the behaviour we're after is if someone passes a file in then we'd want it to raise.